src/osal.h

```diff
@@ -238,6 +333,26 @@ void os_get_button(
    uint16_t                id,         /* Starting from 0 */
    bool                    *p_pressed);
 
+
```

Comment:
These are not really functions that abstract from OS details? The implementation would be the same for all OS:es.

Reply:
Updated osal functions